src/v0/gateway.ts

```diff
@@ -37,6 +54,7 @@ export function createGateway({
     try {
       resp = await fetch(`${baseUrl()}${path}`, options);
     } catch (e) {
+      // TODO: Let's figure out if we can safely swallow this.
       if (e.name === 'AbortError') {
         throw e;
       }
```

Comment:
personally i think it can just be ignored. @dangodev might have some thoughts

Reply:
I have some reasons why I'm not so sure so I think we should figure it out separately, since the consuming code is handling it fine for now.